### ISSUE TITLE
`add_2_calendar` does not specify a Swift version 

### ISSUE THREAD

**Issue by envyram** [ja2375/add_2_calendar#21]
Hi, I get the below error when including this package in pubspec

`add_2_calendar` does not specify a Swift version and none of the targets (`Runner`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod

**Comment by excogitatr** [comment]
Got the same error
```
[!] Unable to determine Swift version for the following pods:

- `add_2_calendar` does not specify a Swift version and none of the targets (`Runner`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.
```

**Comment by TheAlphamerc** [comment]
Got this error while installing  **Pod** in iOS project 

**Steps to produce error**
1. Add **_add_2_calendar_** pub in pubspec file.
2. Open terminal navigates to iOS folder.
3. install **Pod** in ios project by run `pod install` command.

`- add_2_calendar does not specify a Swift version and none of the targets (Runner) integrating it have the SWIFT_VERSION attribute set. Please contact the author or set the SWIFT_VERSION attribute in at least one of the targets that integrate this pod.`

**Comment by a6delali** [comment]
in .podspec file, we can explicitly specify the swift version like this:
s.swift_version = '4.0'
So this way, when the pod is installed, Cocoapods will respect the version choice from podspec.
please refer to #25

**Comment by ja2375** [comment]
Merged #25
